amf-webapi/shared/src/main/scala/amf/plugins/document/webapi/parser/spec/domain/OasResponseParser.scala

```diff
@@ -100,10 +100,12 @@ case class OasResponseParser(map: YMap, adopted: Response => Unit)(implicit ctx:
 
         // OAS 3.0.0
         if (ctx.syntax == Oas3Syntax) {
-          map.key(
-            "content",
-            entry => payloads ++= OasContentsParser(entry, res.withPayload).parse()
-          )
+          map
+            .key("content")
+            .map(entry => {
+              res.set(ResponseModel.Payloads, AmfArray(payloads, Annotations(entry)))
+              payloads ++= OasContentsParser(entry, res.withPayload).parse()
+            })
 
           map.key(
             "links",
```

Comment:
[Out of scope] - The commented code on line 129 should be deleted. Intellij won't let me comment the right line

Reply:
Removed